jupyterhub/values.yaml

```diff
@@ -0,0 +1,27 @@
+singleuser:
+  defaultUrl: "/lab"
+  image:
+    name: pachyderm/notebooks-user
+    tag: v2.4.2
+  extraEnv:
+    "SIDECAR_MODE": "True"
+  extraContainers:
+    - name: mount-server-manager
+      image: pachyderm/mount-server:2.4.2
```

Comment:
Might help to have a comment here so that the user that reads line 29 of the docs knows exactly what you're talking about.  That said, it is pretty obvious :)